.github/workflows/async-icq.yml

```diff
@@ -127,7 +127,7 @@ jobs:
         run: |
           docker image load -i ${{ env.HOST_TAR_PATH }}
           docker image load -i ${{ env.CONTROLLER_TAR_PATH }}
-          docker image load -i testing/previous_images/icq-host_7_0_0.tar
+          docker image load -i testing/previous_images/icq-host_8_0_0.tar
```

Comment:
this should use image async-icq `v7.0.0` still as the base. This way we confirm SDK v47 -> SDK v50 migration works as expected for the module. Same for `baseChain` in `e2e/upgrade_test.go`

modules/async-icq/testing/simapp/upgrades/upgrades.go still uses the baseapp migration logic. We moved this forward to v7.1.1, so if we can confirm v7.0.0->v8.0.0 ICQ upgrade works that covers both upgrade paths